### DISCUSSION TITLE
Husky requires Git >=2.13.0. Got v2.11.0.

### DISCUSSION THREAD

**Discussion by AlanGreyjoy** [saleor/saleor#8087] [Deployment]
I am trying to install on ubuntu, and I keep getting the error
Husky requires Git >=2.13.0. Got v2.11.0.

git --version returns 2.25.1

Any advice?

**Comment by karolkielecki** [comment]
@AlanGreyjoy Hi, I'm not working on Ubuntu, but there are some solutions provided in the following threads:
- https://github.com/typicode/husky/issues/822
- https://stackoverflow.com/questions/65411969/can-we-update-git-version-in-the-docker-image

Good luck :)
